src/powershell-win-core/powershell-win-core.csproj

```diff
@@ -15,7 +15,7 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </Content>
-    <Content Include="..\Modules\Windows-Core\**\*;..\Modules\Windows-Core+Full\**\*;..\Modules\Shared\**\*">
+    <Content Include="..\Modules\Windows\**\*;..\Modules\Shared\**\*">
```

Comment:
Ah, maybe we merge powershell-unix.csproj and powershell-win-core.csproj  in one powershell-core.csproj to make file structure more simple?